<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C</class>
 <widget class="QWidget" name="C">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="propertyNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Property Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="housesOwnedlabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Houses Owned:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="housesOwnedSpinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="hotelOwnedCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Hotel Owned:  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="opponentLandedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Opponent Landed</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentRentlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Rent:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentRentAmountlabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$0</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalProfitLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Profit: </string>
   </property>
  </widget>
  <widget class="QLabel" name="totalProfitAmountLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$0</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>propertyNameLabel</zorder>
  <zorder>housesOwnedlabel</zorder>
  <zorder>housesOwnedSpinBox</zorder>
  <zorder>hotelOwnedCheckBox</zorder>
  <zorder>opponentLandedButton</zorder>
  <zorder>currentRentlabel</zorder>
  <zorder>currentRentAmountlabel</zorder>
  <zorder>totalProfitLabel</zorder>
  <zorder>totalProfitAmountLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
